draft-ietf-opsec-probe-attribution.md

```diff
@@ -177,15 +188,15 @@ When the measurement allows for it, a probe description URI should be included i
 
 - for a {{!RFC792}} ICMPv4 echo request: in the optional data;
 
-- for a {{!RFC768}} UDP datagram: in the data part;
+- for a {{!RFC768}} UDP datagram: in the data part. Note that if the probe is destined to a listened-to/well-known UDP port, the inclusion of the probe description URI may be undefined;
```

Comment:
s/the inclusion of the probe description URI may be undefined;/the inclusion of the probe description URI may produce undefined results;/